src/blob_detection.cpp

```diff
@@ -142,6 +142,43 @@ namespace Blob_Detection
     {
         return _contourX.size() == _edgeX.size();
     }
+    
+    double BlobInfo::_getLengthFromContour( const std::vector < uint32_t > & contourX, const std::vector < uint32_t > & contourY, double & length, Point2d & start, Point2d & end ) 
```

Comment:
@FrankJWH ,  it is slightly different from what I proposed in original issue as
```cpp
double getLengthFromCountour( const std::vector < uint32_t > & contourX, const std::vector < uint32_t > & contourY, double & length Point2DBase< uint32_t > & start, Point2DBase< uint32_t > & end )
```
Could you please explain why such change is more suitable? :)